iron-form-element-behavior.html

```diff
@@ -52,22 +52,20 @@
       },
 
       /**
-       * Need to keep a reference to the form this element is registered
-       * to, so that it can unregister if detached.
+       * The form that the element is registered to.
        */
-      _parentForm: {
+      parentForm: {
```

Comment:
Might be ok to keep it private since it's only used to coordinate between this behavior and the form behavior?